solaris/solaris10/README.md

```diff
@@ -23,18 +23,16 @@ To build a Solaris 10 package, you need to download this repository and use the
 
     Usage: ./generate_wazuh_packages.sh [OPTIONS]
 
-    USAGE: Command line arguments available:
-        -h, --help               Displays this help.
-        -d, --download           Download source file and prepares source directories.
-        -u, --utils              Download and install all dependencies.
-        -b, --build              Build Solaris 10 packages.
-        -c, --clean              Clean all. Even installation files.
+        -b, --branch <branch>               Select Git branch or tag e.g. master
+        -e, --environment                   Install all the packages necessaries to build the pkg package
+        -s, --store  <pkg_directory>        Directory to store the resulting pkg package. By default: /tmp/build
```

Comment:
The default path has changed?.

Reply:
Done: https://github.com/wazuh/wazuh-packages/commit/e1d2b0f75c918f86466d58539581845dbb7a3b75